app/src/main/res/layout/content_tab_coupon.xml

```diff
@@ -31,7 +31,7 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/content_banners"/>
+            app:layout_constraintTop_toBottomOf="@id/content_banners "/>
```

Comment:
Space was inserted in the wrong location. wonder why it can be compiled? However, lint catches it.